comments.php

```diff
@@ -65,7 +65,7 @@
 
 	endif;
 
-	if ( ! comments_open() && '0' != get_comments_number() && post_type_supports( get_post_type(), 'comments' ) ) :
+	if ( ! comments_open() && 0 !== get_comments_number() && post_type_supports( get_post_type(), 'comments' ) ) :
```

Comment:
`get_comments_number()` return `integer` only when is `0`, not a problem in this context, but just leaving this as a side note.